mealplanner-ui/src/pages/MealPlans/DuplicateMealPlan.tsx

```diff
@@ -34,13 +33,12 @@ mutation DuplicateMealPlanMutation($connections: [ID!]!, $mealPlanId: BigInt!, $
 `;
 
 
-export const duplicateMealPlan = (connection: string, id:string, p_id?:string) => {
+export const duplicateMealPlan = (connection: string, id:string) => {
     commitMutation(environment, {
       mutation: duplicateMealPlanGQL,
       variables: {
         connections: [connection],
         mealPlanId: id.toString(),
-        personId: p_id?.toString(),
```

Comment:
You cannot delete it as the mutation will fail for the current person. You need to get the current person id and pass it on here.